lib/SILOptimizer/Transforms/SILMem2Reg.cpp

```diff
@@ -2158,7 +2159,22 @@ void MemoryToRegisters::collectStoredValues(AllocStackInst *asi,
 }
 
 void MemoryToRegisters::canonicalizeValueLifetimes(
-    StackList<SILValue> &owned, StackList<SILValue> &guaranteed) {
+    StackList<SILValue> &owned, StackList<SILValue> &guaranteed,
+    BasicBlockSetVector &livePhiBlocks) {
+  for (auto *block : livePhiBlocks) {
+    SILPhiArgument *argument =
+        cast<SILPhiArgument>(block->getArgument(block->getNumArguments() - 1));
```

Comment:
This is slightly mysterious. You may want to explain that, each new phi generated by Mem2Reg of alloc_stack will be the only new phi in a block at the end of the block's arguments list.

Reply:
Done.